data/aurora/VTBD.lartcc

```diff
@@ -0,0 +1,249 @@
+T;VTBD_APP;N013.55.39.057;E100.15.47.083;
+T;VTBD_APP;N013.55.09.813;E100.15.46.298;
+T;VTBD_APP;N013.54.40.560;E100.15.46.241;
+T;VTBD_APP;N013.54.11.314;E100.15.46.910;
+T;VTBD_APP;N013.53.42.093;E100.15.48.305;
+T;VTBD_APP;N013.53.12.913;E100.15.50.426;
+T;VTBD_APP;N013.52.43.792;E100.15.53.271;
+T;VTBD_APP;N013.52.14.747;E100.15.56.839;
+T;VTBD_APP;N013.51.45.795;E100.16.01.126;
+T;VTBD_APP;N013.51.16.954;E100.16.06.132;
+T;VTBD_APP;N013.50.48.239;E100.16.11.851;
+T;VTBD_APP;N013.50.19.668;E100.16.18.282;
+T;VTBD_APP;N013.49.51.259;E100.16.25.420;
+T;VTBD_APP;N013.49.23.027;E100.16.33.262;
+T;VTBD_APP;N013.48.54.989;E100.16.41.801;
+T;VTBD_APP;N013.48.27.161;E100.16.51.034;
+T;VTBD_APP;N013.47.59.561;E100.17.00.954;
+T;VTBD_APP;N013.47.32.204;E100.17.11.557;
+T;VTBD_APP;N013.47.05.107;E100.17.22.834;
+T;VTBD_APP;N013.46.38.285;E100.17.34.781;
+T;VTBD_APP;N013.46.11.754;E100.17.47.389;
+T;VTBD_APP;N013.45.45.529;E100.18.00.652;
+T;VTBD_APP;N013.45.19.627;E100.18.14.561;
+T;VTBD_APP;N013.44.54.062;E100.18.29.108;
+T;VTBD_APP;N013.44.28.849;E100.18.44.285;
+T;VTBD_APP;N013.44.04.004;E100.19.00.082;
+T;VTBD_APP;N013.43.39.541;E100.19.16.491;
+T;VTBD_APP;N013.43.15.473;E100.19.33.501;
+T;VTBD_APP;N013.42.51.816;E100.19.51.102;
+T;VTBD_APP;N013.42.28.584;E100.20.09.285;
+T;VTBD_APP;N013.42.05.789;E100.20.28.038;
+T;VTBD_APP;N013.41.43.445;E100.20.47.350;
+T;VTBD_APP;N013.41.21.566;E100.21.07.211;
+T;VTBD_APP;N013.41.00.164;E100.21.27.607;
+T;VTBD_APP;N013.40.39.252;E100.21.48.528;
+T;VTBD_APP;N013.40.18.842;E100.22.09.960;
+T;VTBD_APP;N013.39.58.946;E100.22.31.892;
+T;VTBD_APP;N013.39.39.576;E100.22.54.310;
+T;VTBD_APP;N013.39.20.743;E100.23.17.202;
+T;VTBD_APP;N013.39.02.458;E100.23.40.553;
+T;VTBD_APP;N013.38.44.733;E100.24.04.349;
+T;VTBD_APP;N013.38.27.576;E100.24.28.578;
+T;VTBD_APP;N013.38.10.999;E100.24.53.225;
+T;VTBD_APP;N013.37.55.012;E100.25.18.274;
+T;VTBD_APP;N013.37.39.623;E100.25.43.712;
+T;VTBD_APP;N013.37.24.841;E100.26.09.524;
+T;VTBD_APP;N013.37.10.676;E100.26.35.694;
+T;VTBD_APP;N013.36.57.135;E100.27.02.206;
+T;VTBD_APP;N013.36.44.227;E100.27.29.047;
+T;VTBD_APP;N013.36.31.959;E100.27.56.198;
+T;VTBD_APP;N013.36.20.339;E100.28.23.646;
+T;VTBD_APP;N013.36.09.372;E100.28.51.373;
+T;VTBD_APP;N013.35.59.066;E100.29.19.363;
+T;VTBD_APP;N013.35.49.427;E100.29.47.600;
+T;VTBD_APP;N013.35.40.460;E100.30.16.067;
+T;VTBD_APP;N013.35.32.170;E100.30.44.748;
+T;VTBD_APP;N013.35.24.563;E100.31.13.626;
+T;VTBD_APP;N013.35.17.643;E100.31.42.683;
+T;VTBD_APP;N013.35.11.414;E100.32.11.903;
+T;VTBD_APP;N013.35.05.879;E100.32.41.269;
+T;VTBD_APP;N013.35.01.042;E100.33.10.763;
+T;VTBD_APP;N013.34.56.906;E100.33.40.367;
+T;VTBD_APP;N013.34.53.473;E100.34.10.066;
+T;VTBD_APP;N013.34.50.745;E100.34.39.841;
+T;VTBD_APP;N013.34.48.724;E100.35.09.674;
+T;VTBD_APP;N013.34.47.410;E100.35.39.549;
+T;VTBD_APP;N013.34.46.805;E100.36.09.448;
+T;VTBD_APP;N013.34.46.910;E100.36.39.352;
+T;VTBD_APP;N013.53.36.800;E100.35.46.300;

```

Comment:
What is this airspace?

Reply:
This is the Don Mueang Approach, needs to be in the VTBD.lartcc files but this is the imaginary line where the Don Mueang Approach is suppose to be at.